rust/kernel/print.rs

```diff
@@ -189,7 +189,7 @@ macro_rules! print_macro (
 // well as playing with the `doc` attribute. Furthermore, they cannot be easily
 // imported in the prelude due to [1]. So, for the moment, we just write them
 // manually, like in the C side; while keeping most of the logic in another
-// macro, i.e. [`print_macro`].
+// macro, i.e. `print_macro`.
```

Comment:
Sorry, which doc update? Do you mean the coding guidelines one I suggested? Yeah, it could be the same commit (or not). But if you are going to do the guidelines change, then the important bit is that part, since we are adding a rule, so I would probably submit that change (actual removal and guideline change) independently of this patch series.